resources/views/customer/customer-show.twig

```diff
@@ -0,0 +1,98 @@
+{% extends "base.twig" %}
+
+{% block body %}
+<h1>{{ customer.company_name }}</h1>
+<form class="form-horizontal">
+    <div class="row">
+        <label class="control-label col-xs-8 col-sm-2">
+            VAT number
+        </label>
```

Comment:
Donc si je comprends bien, il faudrait que je mette un `input` dans chacun de mes paragraphe avec des attributs `name` et `id` et que j'ajoute un attribut `for` aux label?
Du coup ça donnerait des trucs du style:
```twig
<div class="row">
        <label class="control-label col-xs-8 col-sm-2" for="vat_number">
            VAT number
        </label>
        <div class="col-xs-4 col-sm-2">
            <p class="form-control-static">
                <input name="vat_number" value="{{ customer.vat }}" disabled>
            </p>
        </div>
    </div>
```
Du coup l'affichage est moins sympa mais si ça rend l'application plus accessible ça me va.